pimp-it.lua

```diff
@@ -0,0 +1,1060 @@
+-- Optimize dwarves for fort-mode work. Pimp your dwarves, make your life easier in managing labours.
+-- written by josh cooper(cppcooper) [created: 12-2017 | last edited: 10-2018]
+--[====[
+pimp-it
+=======
+Optimize dwarves for fort-mode work.
+The core function takes dwarves and allocates a "dorf_job" to each dwarf.
+This decision takes into account current counts for each dorf_job, in
+relation to how many should be allocated based on the working population
+size.
+
+Usage: ``pimp-it -help`` or ``pimp-it -select <sel-opt> -<command> <args>``
+
+:help:               Highly detailed help documentation.
+:select <option>:    Indicates the next parameter will be indicate which dwarves to select
+]====]
+
+print("v1.1")
+utils ={}
+utils = require('utils')
+json = require('json')
+local rng = require('plugins.cxxrandom')
+local engineID = rng.MakeNewEngine()
+print("Loading data tables..")
+local dorf_tables = dfhack.script_environment('dorf_tables')
```

Comment:
```suggestion
local dorf_tables = reqscript('dorf_tables')
```
This will work with the `--@module = true` above

Reply:
done, this is in the next commit